data/templates/metronome/share.tpl.php

```diff
@@ -0,0 +1,189 @@
+<?php
```

Comment:
Do you know if equivalent script exist in python (we would avoid to add php in dependence of yunohost just for xmpp upload feature

Reply:
This should not be complicated to rewrite it in python. I just took the official metronome example script. I can do it when I have time

Reply:
Could we iterate instead of having the perfect PR ? Because this PR is quite useful.